source/features/bugs-tab.tsx

```diff
@@ -39,7 +39,10 @@ async function init(): Promise<void | false> {
 		return false;
 	}
 
-	const issuesTab = (await elementReady('.pagehead [data-hotkey="g i"]'))?.parentElement;
+	const issuesTab = (await elementReady([
+		'.pagehead [data-hotkey="g i"]',
+		'.js-repo-nav [data-hotkey="g i"]' // "Repository refresh" layout
+	].join()))?.parentElement;
```

Comment:
Even if it doesn't work on GHE, it's fine
```suggestion
	const issuesTab = (await elementReady('.js-repo-nav [data-hotkey="g i"]'))?.parentElement;
```

Reply:
If you feel like it, you can drop `.parentElement` here
```suggestion
	const issuesLink = await elementReady('.js-repo-nav [data-hotkey="g i"]'));
```

And move it to:

```ts
	const bugsTab = issuesTab.parentElement!.cloneNode(true);
```

Reply:
This exists in GHE with the new selector. If there are specific concerns, @ me and I can check.